integrations/transportation/tripit.js

```diff
@@ -148,6 +166,69 @@ async function fetchRail(modifiedSince, isPast = true, logger) {
   return { activities: flatten(activities), modifiedSince: data.timestamp };
 }
 
+async function fetchLodging(modifiedSince, isPast = true, logger) {
+  const url = modifiedSince
+    ? `/v1/list/object/type/lodging/past/${isPast}/format/json/modified_since/${modifiedSince}`
+    : `/v1/list/object/type/lodging/past/${isPast}/format/json`;
+  const res = await manager.fetch(url);
+  if (!res.ok) {
+    const text = await res.text();
+    throw new HTTPError(text, res.status);
+  }
+  const data = await res.json();
+  const pageNum = parseInt(data['page_num'], 10);
```

Comment:
There's a lot of square-bracket-notations, but our eslint config wants dot-nation - a quick `eslint --fix integrations/transportation/tripit.js` should do it :)

Reply:
I'd rather leave it as it is to avoid changing everything (also because it's just a warning).
Not sure I like that rule also ;-P

Reply:
Haha okay, guess it's a matter of personal opinion - but let's just leave it for now ;)